### PR TITLE
Add new logic

### PR DESCRIPTION
**Pull request by xelabik**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -5,5 +5,7 @@ def print_hi(name):
     print(f"Hi, {name}") 
 
 if __name__ == "__main__":
-    print_hi("Alex")
+    friends = ["Dima", "Demon"]
+    print_hi(name=",".join(friends))
 
+# added my changes
```
Comment (dimaka-wix) [inline test.py]: Please remove the comments

File: test.py [hunk 2]
```diff
@@ -5,5 +5,7 @@ def print_hi(name):
     print(f"Hi, {name}")
```
Comment (dimaka-wix) [inline test.py:5]: Please add types

**Review by dimaka-wix — CHANGES_REQUESTED** [review]

File: test.py (hunk 1 above)
Comment (xelabik) [reply test.py]: done

File: test.py (hunk 2 above)
Comment (xelabik) [reply test.py:5]: done

**Review by dimaka-wix — APPROVED** [review]
# Good job

**Pull request merged**